src/rgw/rgw_file.h

```diff
@@ -802,7 +802,10 @@ namespace rgw {
 			     const uint32_t flags = RGWFileHandle::FLAG_NONE) {
       using std::get;
 
-      LookupFHResult fhr { nullptr, RGWFileHandle::FLAG_NONE };
+      // This seems like a Clang compiler error.
+      // It result in irgw::RGWFileHandle::FLAG_NONE being undefined during linking.
+      // LookupFHResult fhr { nullptr, RGWFileHandle::FLAG_NONE };
+      LookupFHResult fhr { nullptr, 0 };
```

Comment:
@tchaikov 
Works like a charm. No more complaints in the linker.
I will change and rebase this commit.